provision/provision_validate_test.go

```diff
@@ -1,13 +1,334 @@
-package provision_test
+package provision
```

Comment:
@crgisch could you add the copyright headers?

Reply:
yes 💯 
done